test/sysv_sliding_window_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class TestSysVSlidingWindow < MiniTest::Unit::TestCase
+  CLASS = ::Semian::SysV::SlidingWindow
+
+  def setup
+    @sliding_window = CLASS.new(max_size: 6,
+                                name: 'TestSysVSlidingWindow',
+                                permissions: 0660)
+    @sliding_window.clear
+  end
+
+  def teardown
+    @sliding_window.destroy
+  end
+
+  include TestSimpleSlidingWindow::SlidingWindowTestCases
```

Comment:
I didn't add the shared tests in the commit, the tests and build would fail. Do you want me to include it?